flink-table/flink-table-planner-blink/src/main/java/org/apache/flink/table/planner/delegation/ExecutorBase.java

```diff
@@ -43,25 +42,16 @@ public ExecutorBase(StreamExecutionEnvironment executionEnvironment) {
 		this.executionEnvironment = executionEnvironment;
 	}
 
-	public void setTableConfig(TableConfig tableConfig) {
-		this.tableConfig = tableConfig;
-	}
-
 	public StreamExecutionEnvironment getExecutionEnvironment() {
 		return executionEnvironment;
 	}
 
 	@Override
-	public void apply(List<Transformation<?>> transformations) {
-		transformations.forEach(getExecutionEnvironment()::addOperator);
+	public JobExecutionResult execute(Pipeline pipeline) throws Exception {
+		return executionEnvironment.execute((StreamGraph) pipeline);
 	}
 
-	/**
-	 * Translates the transformations applied into this executor to a stream graph.
-	 */
-	public abstract StreamGraph getStreamGraph(String jobName);
-
-	protected String getNonEmptyJobName(String jobName) {
+	String getNonEmptyJobName(String jobName) {
```

Comment:
Why shall we change visibility? If for `VisitingForTesting`, annotate it.

Reply:
I will revert this